### PR TITLE
React to main repo updates

### PR DESCRIPTION
**Pull request by StachuDotNet**
This repo will tidy up a bit once #302 is merged.

### CODE REVIEW

File: docs/contributing/rescript-and-fsharp-for-dark-developers.md [hunk 1]
```diff
@@ -750,7 +750,7 @@ ported to what is now ReScript in 2018. ReScript is the new name (as of 2020)
 for what was sometimes called Bucklescript and sometimes called ReasonML. You
 may see references to Bucklescript in our codebase (including the prefix "bs").
 
-Our frontend used an alternate syntax (.ml, as ReScript is based on OCaml and we
-used OCaml on our backend too). We switched over to .res in 2021 - however, a
+Our frontend used an alternate syntax (.TODO, as ReScript is based on OCaml and
```
Comment (pbiggar) [inline docs/contributing/rescript-and-fsharp-for-dark-developers.md:753]: Alternate syntax isn't correct anymore.

File: docs/contributing/rescript-and-fsharp-for-dark-developers.md [hunk 2]
```diff
@@ -115,7 +115,7 @@ returns the result of that expression.
 
 Most of the code in Dark uses
 [Tablecloth](https://github.com/darklang/tablecloth), which has the same
-[interface](https://github.com/darklang/tablecloth/blob/main/rescript/src/Tablecloth.ml)
+[interface](https://github.com/darklang/tablecloth/blob/main/rescript/src/Tablecloth.TODO)
```
Comment (pbiggar) [inline docs/contributing/rescript-and-fsharp-for-dark-developers.md:118]: Maybe just unlink the interface?

**Comment by pbiggar** [comment]
Looks great! Feel free to merge once to TODOs are done and I can review after.

File: docs/contributing/debugging.md [hunk 3]
```diff
@@ -4,31 +4,32 @@ title: Debugging
...
 
-Both the frontend and backend have simple log statements, which return `()`.
+### Client
 
-- Frontend (prints in the browser console): `Js.log2 "my message" anyValue`
-- Backend (prints in `rundir/logs/server.log` in purple; search for
-  **"INSPECT"**): `Log.inspecT "my message" any_value`
+In the `client`, you may log with either of:
 
-If you have existing code like this:
```
Comment (StachuDotNet) [inline docs/contributing/debugging.md:13]: TODO: fix this formatting a bit. (preview and make sure all is OK)

File: docs/contributing/debugging.md [hunk 4]
```diff
@@ -4,31 +4,32 @@ title: Debugging
...
+- `Js.log2("my message", anyValue)`
+- `Debug.loG("my message", anyValue)` Both of these print in the browser
```
Comment (StachuDotNet) [inline docs/contributing/debugging.md:15]: I see both of these in use but I've only used the latter - not sure why.
Is there any intention to migrate Js.log2 usages to fns in the `Debug` module? Or when should someone use one over the over?

Comment (pbiggar) [reply docs/contributing/debugging.md:15]: Debug.log is more flexible, no plans but no reasons to stick with log2 afaik

**Pull request merged**